src/private/__tests__/actions.test.js

```diff
@@ -11,6 +11,11 @@ describe('actions', () => {
       state = newState
       store.state = newState
     },
+    resetState: obj => {
+      const newState = { ...state, ...obj }
+      state = newState
```

Comment:
The reset state would transform it into an empty object, is that right?
```suggestion
      state = {}
```

Reply:
Yep, changed it :)